common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -801,6 +807,29 @@ def answer_problem(self, correctness):
         self.problem_page.fill_answer(second_addend, input_num=1)
 
 
+class JSInputTypeTest(ProblemTypeTestBase, ProblemTypeA11yTestMixin):
```

Comment:
Runs the a11y test point.

![image](https://cloud.githubusercontent.com/assets/484484/20770132/a2bfde3a-b712-11e6-880c-a854312e9150.png)

Before code changes, this a11y test failed both for no title being present, and the section element with no header.